styles.css

```diff
@@ -16,7 +16,11 @@ table {
     By default, when you set borders on table elements, they will look pretty ugly.
      */
     border-collapse: collapse;
-    border: 3px solid purple;
+    border: 3px solid lightblue;
+}
+
+tr:nth-child(1) {
```

Comment:
Unnecessary duplication of ‘tr:nth-child(1)’ node. This node already exists on line 47 in the same file.